nodejs/week2/src/backend/routes/meals.js

```diff
@@ -0,0 +1,54 @@
+//Bringing Dependencies
+const express = require('express');
+const app = express();
+const meals = require("../data/meals");
+const reviews = require("../data/reviews");
+
+//JSON response for with corresponding ID
+app.get('/meals/:id', (req,res)=>{
+    const id = Number(req.params.id);
+    const mealId = meals.filter(meal=>meal.id === id);
+    res.send(mealId);
+    if(mealId.length === 0){
+        res.status(401).send('Not found meal');
+}
+    else{
+        res.send(mealId);
+    }
+    
+});
+
+//JSON response for meals that has a price smaller than maxPrice
+app.get('/meals', (req,res)=>{
+    if(req.query.maxPrice){
+        const maxPrice = parseInt(req.params.maxPrice);
```

Comment:
what happens if maxPrice is not a valid number?

What happens if all the other params aren't valid?